pennylane/math/__init__.py

```diff
@@ -36,17 +36,26 @@
 """
 from .fn import (
     T,
+    abs_ as abs,
```

Comment:
Yep, I though it best not to redefine `abs` and `sum` _directly_ in the `fn` module, since they might need to be used.

I'm doing the redefining here on import instead, since the `__init__` file doesn't contain any processing or logic.